requirements_dev.txt

```diff
@@ -12,10 +12,6 @@ asttokens==2.0.5
     # via stack-data
 backcall==0.2.0
     # via ipython
-backports-zoneinfo==0.2.1
-    # via
-    #   -r requirements.txt
-    #   django
```

Comment:
Did pip-compile remove these lines ?

Reply:
This file was not changed manually.

Instead, it was generated by pip-compile `6.12.2` with Python 3.10.6 (main, Nov 14 2022, 16:10:14) [GCC 11.3.0] on linux:
```
pip-compile requirements_dev.in
```